integration/monai/examples/spleen_ct_segmentation_sim/job_multi_gpu/app/config/config_fed_client.json

```diff
@@ -27,7 +27,6 @@
               "args": {
                 "bundle_root": "config/spleen_ct_segmentation",
                 "local_epochs": 10,
-                "multi_gpu": true,
```

Comment:
why we don't need this anymore?

Reply:
Deprecated in MONAI 1.2. Multi-gpu setup is now done within the Bundle API.